api_app/analyzers_manager/observable_analyzers/checkphish.py

```diff
@@ -0,0 +1,81 @@
+# This file is a part of IntelOwl https://github.com/intelowlproject/IntelOwl
+# See the file 'LICENSE' for copying permission.
+
+import time
+
+import requests
+
+from api_app.analyzers_manager import classes
+from api_app.exceptions import AlreadyFailedJobException, AnalyzerRunException
+from tests.mock_utils import MockResponse, if_mock_connections, patch
+
+
+class CheckPhish(classes.ObservableAnalyzer):
+    base_url: str = "https://developers.checkphish.ai/api/neo/scan"
+    status_url: str = base_url + "/status"
+
+    def set_params(self, params):
+        self.polling_tries = params.get("polling_tries")
+        self.polling_time = params.get("polling_time")
```

Comment:
if, for any chance, there is someone who changed the `analyzer_config.json`, those values would be `None`.

Please just put a default here too and then the PR is completed. TY